styhead/rz-cmn: rebase the latest changes from the styhead/rz-sbc branch

tiennguyenuh opened the pull request:
This PR includes:
- Set the default tune to cortexa55 instead of armv8a for optimization on the RZ/G2L's Cortex-A55 core
- Add docker.cfg to the Linux Yocto recipe
- Add Ethernet patch to prevent eth0 from being disabled before eth1.
- Add `package-management` and `package_deb` to support APT in minimal-based images.
- Restore DNS and time sync services needed for network and date accuracy.
- Enable xwayland support and optimize kernel config

tiennguyenuh committed:
rz-cmn-srp: add audio related-config

Add `CONFIG_SND_ALOOP` to the kernel config fragment

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: add Ethernet patch fix

Add Ethernet patch to prevent eth0 from being disabled before eth1.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: enable APT support for minimal-based images

Add `package-management` and `package_deb` to support APT in
minimal-based images.

Weston-based images worked previously because these features are included by
default.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: retain systemd-resolved and timesyncd in quickboot images

Restore DNS and time sync services needed for network and date accuracy.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: linux recipe and configuration

Changes include:
	- Set the default tune to cortexa55 instead of armv8a for
	  optimization on the RZ/G2L's Cortex-A55 core
	- Add docker.cfg to the Linux Yocto recipe whe

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: Update README and add default BB_PRESSURE_MAX_MEMORY setting

Changes include:
	- Add BB_PRESSURE_MAX_MEMORY with a default value of `100000` in
	  local.conf
	- Add a note to inform users that this value should be adjusted
	  based on their system resources
	- Update README for new cortexa55 tune

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: add tkinter to local.conf

Enable tkinter installation for python3

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: modify v2h support patch and add config fragment

 Changes include:
 	- Use renesas_defconfig instead of defconfig
	- Add config for laird modules
	- Modify the patch to support V2H as it not up
	  to date.
	- Organize patches and configs by board for better clarity

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: enable xwayland support and optimize kernel config

Changes include:
	- Add 'xwayland=true' to the [core] section of weston.ini to
	  enable automatic Xwayland startup
	- Add optimize kernel config fragment

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

vudangRVC commented on recipes-kernel/linux/linux-yocto_6.10.bbappend:
```diff
@@ -58,28 +76,28 @@ DEVICETREE_NAME:rz-cmn = "rzg2l-sbc r9a07g044l2-smarc r9a07g054l2-smarc r9a09g05
 KERNEL_DEVICETREE:rz-cmn = "${@' '.join(['renesas/%s.dtb' % devicetree_name for devicetree_name in d.getVar('DEVICETREE_NAME').split()])}"
```
There will be some device tree overlays for the RZG2L-EVK and RZV2L-EVK. Please implement a mechanism to support this

tiennguyenuh replied:
Updated in the commit https://github.com/Renesas-SST/meta-renesas/commit/ca8935ce5e6d8994fc88c048cb09f9ec379fdc1d, new dtbs for these cameras were added to the DEVICETREE_NAME variable.

tiennguyenuh committed:
rz-cmn-srp: kernel: add support camera ov5645

Changes include:
	- Add kernel patch to support the camera OV5645
	- Deploy corresponding device tree blob to the boot partition

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: recipes-kernel: drop unused config in renesas_defconfig

These kernel options are mainly used for Docker and container networking,
including connection tracking and packet filtering. They are not needed
for non-container use cases and have been removed to simplify the kernel
config

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: recipes-kernel: add patch support and config fragments

Changes include:
	- Applied kernel patches to enable additional features on
	  RZ/V2H-EVK and RZ/G2L-EVK boards.
	- Integrated necessary config fragments into renesas_defconfig
	  for proper hardware and feature enablement.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>
